pkg/alertmanager/alertmanager.go

```diff
@@ -356,7 +356,7 @@ func (am *Alertmanager) ApplyConfig(userID string, conf *config.Config, rawCfg s
 		return notifier
 	})
 	if err != nil {
-		return nil
+		return err
```

Comment:
Please undo this change, since it diverges from the PR topic.
```suggestion
		return nil
```